www/front_src/src/Resources/translatedLabels.ts

```diff
@@ -249,3 +249,6 @@ export const labelNoContactIsConfiguredForThisResource =
   'No contacts are configured for this resource';
 export const labelNoContactGroupsIsConfiguredForThisResource =
   'No contact groups are configured for this resource';
+export const labelCategory = 'Categories';
```

Comment:
There's a incoherence here: do you want the label to be plural or singular? if plural is the answer, then the name should be labelCategories

Reply:
yes :/ it's fixed